pkg/cnab/provider/invoke.go

```diff
@@ -4,25 +4,51 @@ import (
 	"fmt"
 
 	cnabaction "github.com/deislabs/cnab-go/action"
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/cnab-go/claim"
 	"github.com/hashicorp/go-multierror"
 	"github.com/pkg/errors"
 )
 
-func (d *Runtime) Invoke(action string, args ActionArguments) error {
-	c, err := d.instanceStorage.Read(args.Claim)
+func (d *Runtime) getClaim(bun *bundle.Bundle, actionName, claimName string) (*claim.Claim, bool, error) {
```

Comment:
Even though unexported, I wonder if we can write a function description here, especially to clarify the return vals?